client-app/modules/customer-reviews/components/product-reviews.vue

```diff
@@ -45,6 +45,22 @@
           <div>
             {{ review.review }}
           </div>
+
+          <div v-if="review.images?.length" class="image-gallery__thumbs-container">
```

Comment:
According to BEM methodology, an element cannot be used separately from its block.  
Since we are not using the corresponding BEM block here, we should apply TailwindCSS classes instead.  
Please replace the BEM-specific styles with Tailwind utilities.